crates/http/src/routes.rs

```diff
@@ -201,6 +209,20 @@ impl fmt::Display for RoutePattern {
     }
 }
 
+enum Wildness {
```

Comment:
This is cute, but can we think of a clearer name?  I'm not sure what it would be, but I'm pretty sure it isn't this.